packages/flutter/test/material/navigation_drawer_test.dart

```diff
@@ -83,6 +83,45 @@ void main() {
     expect(_getMaterial(tester).color, equals(color));
   });
 
+
+  testWidgetsWithLeakTracking('NavigationDrawer can update destination background color',
+      (WidgetTester tester) async {
+    const Color color = Colors.yellow;
+    final GlobalKey<ScaffoldState> scaffoldKey = GlobalKey<ScaffoldState>();
+    final ThemeData theme= ThemeData.from(colorScheme: const ColorScheme.light());
```

Comment:
nit: I think this can be simplified to just:
```suggestion
    final ThemeData theme = ThemeData();
```